### ISSUE TITLE
RC1 project.json dependencies options: private not working or removed

### ISSUE THREAD

**Issue by FlorianRainer** [aspnet/Home#1126]
Like described here (https://github.com/tdykstra/AspnetWiki/blob/master/Project.json%20file.md#dependencies) there is existing a option `private` on `dependencies` to not show this dependency transitively

i wanted to use it in my N-Tier Layer Application to "hide" the Layer like it is possible in Asp.Net 4.x

Example:
Project1:

```
{
  "version": "1.0.0-*",
  "description": "Layer 1",
  "authors": [ "" ],
  "tags": [ "" ],
  "projectUrl": "",
  "licenseUrl": "",
  "frameworks": {
    "net451": { },
    "dotnet5.4": {
      "dependencies": {
        "Microsoft.CSharp": "4.0.1-beta-23516",
        "System.Collections": "4.0.11-beta-23516",
        "System.Linq": "4.0.1-beta-23516",
        "System.Runtime": "4.0.21-beta-23516",
        "System.Threading": "4.0.11-beta-23516"
      }
    }
  }
}
```

Project2:

```
{
  "version": "1.0.0-*",
  "description": "Layer 2",
  "authors": [ "" ],
  "tags": [ "" ],
  "projectUrl": "",
  "licenseUrl": "",
  "frameworks": {
    "net451": { },
    "dotnet5.4": {
      "dependencies": {
        "Microsoft.CSharp": "4.0.1-beta-23516",
        "System.Collections": "4.0.11-beta-23516",
        "System.Linq": "4.0.1-beta-23516",
        "System.Runtime": "4.0.21-beta-23516",
        "System.Threading": "4.0.11-beta-23516"
      }
    }
  },
  "dependencies": {
    "ProjectLayer1": {
      "version": "1.0.0-*",
      "options": "private"
    }
  }
}
```

Project 3:

```
{
  "version": "1.0.0-*",
  "description": "Layer 3",
  "authors": [ "" ],
  "tags": [ "" ],
  "projectUrl": "",
  "licenseUrl": "",
  "frameworks": {
    "net451": { },
    "dotnet5.4": {
      "dependencies": {
        "Microsoft.CSharp": "4.0.1-beta-23516",
        "System.Collections": "4.0.11-beta-23516",
        "System.Linq": "4.0.1-beta-23516",
        "System.Runtime": "4.0.21-beta-23516",
        "System.Threading": "4.0.11-beta-23516"
      }
    }
  },
  "dependencies": {
    "ProjectLayer2": {
      "version": "1.0.0-*",
      "options": "private"
    }
  }
}
```

Project 1 and 2 are Class Libraries (Package) and Project 3 is a Web Application.
All are in the same solution and connected via project reference

What i expected to see:
Project 3 can not access public members or Project 1

What is happened: Project 3 can access all public members of Project 1.

i noticed that the `options` property of `depenencies` doesn't exist in http://json.schemastore.org/project

is this a removed feature or sould this be done in a differnt way on RC1?

**Comment by davidfowl** [comment]
Private never worked. Those docs are outdated. The feature you want will be implemented when nuget supports the concept (it will in 3.3) in packages

**Comment by FlorianRainer** [comment]
Thanks for the Feedback,
good to know so i will stop searching for a solution and waiting for NuGet 3.3.
Is this feature already implemented in NuGet 3.3.0 Beta?

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
